hack/docker-builder/fedora-virt-preview.repo

```diff
@@ -0,0 +1,24 @@
+# Fedora Virt preview repository
```

Comment:
Why we need this file on this project?

Reply:
With the removed dependencies we don't need it now, removed